README.md

```diff
@@ -108,46 +113,52 @@ __Review Recent Login__: Sometimes Facebook will ask you to review your recent l
-  + `body` - The string corresponding to the message that was just received.
-  + `thread_id` - The thread_id representing the thread in which the message was sent.
-  + `coordinates` - An object containing `latitude`, `longitude`, and `accuracy`.
-  + `type` - The string `"message"`, `"sticker"`, `"file"`, `"photo"`, `"animated_image"`, or `"event"` (if applicable, see below).
-
-If `type` is `"sticker"` there will be a `sticker_id` and `sticker_url` field instead of `body`.
-
-If `type` is `file` there will be a `name` and a `file_url` instead of `body`.
-
-If `type` is `photo` there will be `name`,`hires_url`, `thumbnail_url`, and `preview_url` instead of `body`.
-
-If `type` is `animated_image` there will be `name`, `url`, and `preview_url`  instead of `body`.
-
-If enabled through [setOptions](#setOptions), this will also handle events. In this case, `message` will be either a message (see above) or an event object with the following fields
-- `type` - The string `"event"`
-- `thread_id` - The thread_id representing the thread in which the message was sent.
-- `log_message_type` - String representing the type of event (`"log:thread-name"`, `"log:unsubscribe"`, `"log:subscribe"`, ...)
-- `log_message_data` - Data relevant to the event.
-- `log_message_body` - String printed in the chat.
-- `author` - The person who performed the event.
-
-If enabled through [setOptions](#setOptions), this will also return presence, (`type` will be `"presence"`), which is the online status of the user's friends. The object given to the callback will have the following fields
-- `timestamp` - how old the information is
-- `userId` - the id of the user whose status this packet is describing
-- `statuses` - An object will the following fields: `fbAppStatus`, `messengerStatus`, `otherStatus`, `status` and `webStatus`. All those can contain either of the following values: `"active"`, `"idle"`, `"offline"`.
+  + `senderName`: First and last name of the person who just sent the message.
+  + `senderID`: The id of the person who sent the message in the chat with thread_id.
+  + `participantNames`: An array containing only the first names of the other participants in the thread (sender included).
+  + `participantIDs`: An array containing the ids of everyone in the thread (sender included).
+  + `body`: The string corresponding to the message that was just received.
+  + `threadID`: The thread_id representing the thread in which the message was sent.
+  + `threadName`: The name of the receiver in a single person chat or the name of the group chat.
+  + `location`
+  + `messageID`: A string representing the message ID.
+  + `attachments`: An array of attachments to the message.
+  
+If `attachments` contains an object with type is `"sticker"`, the same object will contain the following fields: `url`, `stickerID`, `packID`, `frameCount`, `frameRate`, `framesPerRow`, `framesPerCol`, `spriteURI`, `spriteURI2x`, `height`, `width`, `caption`, `description`.
```

Comment:
Would it be too lazy to just point at the `formatXXXX` function? We wouldn't have to update the docs, and it wouldn't be any less clear. (This applies to the paragraphs below too)